pkg/controlplane/rest/import.go

```diff
@@ -31,14 +31,22 @@ type importHandler struct {
 }
 
 func toK8SImport(imp *store.Import, namespace string) *v1alpha1.Import {
+	sources := make([]v1alpha1.ImportSource, len(imp.Peers))
+	for i, pr := range imp.Peers {
+		sources[i].Peer = pr
+		sources[i].ExportName = imp.Name
```

Comment:
Do I need always to specify the ExportName and ExportNamespace? what happens if we don't?

Reply:
The `Import` API only includes an array of peers.
Here, we translate this array to `ImportSource`, which requires not just the peer, but also the export name and namespace.
In the old API (which is kept here), the export name must match the import name, and there's no namespaces.